test/player_integration.js

```diff
@@ -87,6 +88,31 @@ describe('Player', () => {
     });
   });  // describe('attach')
 
+  describe('set mediaElement.currentTime in manifestparsed', () => {
+    it('does not get segments prior to mediaElement.currentTime', async () => {
+      // if (shaka.util.Platform.isApple()) {
+      //   pending('setting currentTime during load is not supported by apple');
+      // }
+      player.addEventListener('manifestparsed', () => {
+        player.getMediaElement().currentTime = 24;
+        expect(player.getMediaElement().currentTime).toBe(24);
+      });
+      let requestedFirstMediaSegment = false;
+      player.getNetworkingEngine().registerRequestFilter(
+          (type, request) => {
+            shaka.log.debug('requesting ' + request.uris[0] +
+                ' Range=' + request.headers['Range']);
+            if (request.headers['Range'] === 'bytes=558-326077') {
+              requestedFirstMediaSegment = true;
+            }
+          });
+      await player.load('https://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd');
```

Comment:
We try to avoid using external assets for these tests, and place them behind a flag (`build/test.py --external`) when we must use external assets.

You should copy from tests that use simulated assets through special networking plugins in the test environment.  This allows for reliable tests without a good internet connection, so we can avoid network-based flakiness.

For example, `test:sintel_compiled` should work here as a manifest URL.  You'll find that this is backed by the code in `test/test/util/test_scheme.js` and `test/test/util/stream_generator.js`, and that the first segment URL for video will be something like `test:sintel_compiled/video/1`, which you can check for in your filter above.